docker/llm/finetune/qlora/xpu/docker/Dockerfile

```diff
@@ -34,6 +34,9 @@ RUN curl -fsSL https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-P
     pip install git+https://github.com/huggingface/transformers.git@${TRANSFORMERS_COMMIT_ID} && \
     pip install peft==0.5.0 datasets accelerate==0.23.0 && \
     pip install bitsandbytes scipy && \
+    git clone https://github.com/intel-analytics/BigDL.git && \
+    mv BigDL/python/llm/example/GPU/LLM-Finetuning/common /common && \
```

Comment:
In `qlora_finetuning_cpu.py`, the `common` path is expected to be in `os.path.join(current_dir, '..', '..', 'GPU', 'LLM-Finetuning')`, I wonder whether works if moving to `/common` here. Other LGTM.

Reply:
This is for xpu, it works for the `common_util_path = os.path.join(current_dir, '..', '..').`

Reply:
> This is for xpu, it works for the `common_util_path = os.path.join(current_dir, '..', '..').`

Got it, LGTM

Reply:
> This is for xpu, it works for the `common_util_path = os.path.join(current_dir, '..', '..').`

got it, LGTM